### PR TITLE
prettify code

### PR DESCRIPTION
**Pull request by hrishikeshio**

### CODE REVIEW

File: contracts/Core/BlockManager.sol [hunk 1]
```diff
@@ -79,51 +70,25 @@ contract BlockManager is Initializable, ACL, BlockStorage {
         uint256[] memory medians,
         uint256 iteration,
         uint256 biggestInfluencerId
-    ) public initialized checkEpoch(epoch) checkState(parameters.propose()) {
+    ) external initialized checkEpoch(epoch) checkState(parameters.propose()) {
```
Comment (SkandaBhat) [inline contracts/Core/BlockManager.sol:73]: Is this intended?

File: contracts/Core/StakeManager.sol [hunk 2]
```diff
@@ -276,44 +276,26 @@ contract StakeManager is Initializable, ACL, StakeStorage, Pause {
         initialized
         checkEpoch(epoch)
         checkState(parameters.commit())
-        whenNotPaused()
+        whenNotPaused
```
Comment (SkandaBhat) [inline contracts/Core/StakeManager.sol:279]: Is this expected?

File: contracts/Core/BlockManager.sol (hunk 1 above)
Comment (hrishikeshio) [reply contracts/Core/BlockManager.sol:73]: yes. i dont know how it was set public. maybe erroneous conflict resolution in earlier prs

File: contracts/Core/interface/IBlockManager.sol [hunk 3]
```diff
@@ -37,19 +40,15 @@ interface IBlockManager {
 
     function confirmBlock() external;
 
-    function getBlock(uint256 epoch) external view returns(Structs.Block memory _block);
-    function getBlockMedians(uint256 epoch) external view returns(uint256[] memory _blockMedians);
+    function getBlock(uint256 epoch) external view returns (Structs.Block memory _block);
 
-    function getLowerCutoffs(uint256 epoch) external view returns(uint256[] memory _lowerCutoffs);
+    function getBlockMedians(uint256 epoch) external view returns (uint256[] memory _blockMedians);
```
Comment (SkandaBhat) [inline contracts/Core/interface/IBlockManager.sol:45]: Is this expected?

File: contracts/Core/StakeManager.sol (hunk 2 above)
Comment (hrishikeshio) [reply contracts/Core/StakeManager.sol:279]: ya

Comment (hrishikeshio) [reply contracts/Core/StakeManager.sol:279]: ya

**Pull request merged**